torch/csrc/jit/backends/nnapi/nnapi_backend_lib.cpp

```diff
@@ -0,0 +1,46 @@
+#include <torch/csrc/jit/backends/backend.h>
+#include <torch/csrc/jit/backends/backend_exception.h>
+
+namespace torch {
+namespace jit {
+
+class NnapiBackend : public PyTorchBackendInterface {
```

Comment:
Add comments to explain why this file with no actual implementation is needed. Add TODO task number T91991928.

Reply:
Also, for future PR, this would benefit from some doc about how the overall NNAPI backend works.

Reply:
I'll keep this in mind! Just to clarify, do you mean the NNAPI backend delegation or NNAPI itself?

Reply:
Just the backend: things like what it receives as the execution payload, what compile will do with it, and the same for execute.